src/tools/cephfs/MetaTool.cc

```diff
@@ -773,10 +773,7 @@ int MetaTool::list_meta(meta_op &op)
 }
 unsigned long long MetaTool::conv2hexino(const char* ino)
 {
-  unsigned long long iino = 0;
-  std::stringstream conv;
-  conv << ino;
-  conv >> iino;
+  unsigned long long iino = strtoull (ino, NULL, 0);
```

Comment:
i'd suggest just drop this helper and use `std::stoull()` instead.

Reply:
We will get the decimal format of ino, When problems happen.
but we need hexadecimal format of ino to find it from memory cache, mds dump ino, or from disk using this tool.
That's why I added this method.

Reply:
updated it.
Now this method is only for finding the file from disk.